### ISSUE TITLE
[BUG] Crash at startup after updating to v3.0.0-beta6

### ISSUE THREAD

**Issue by desertproject** [jonjomckay/fritter#594]
After updating from v3.0.0-beta5 to beta6 and opening it, Fritter tries to load some tweets (but can't get any) for a about a second and crashes. No error is shown, only a message from android saying something like "Unfortunately, Fritter has stopped. Open the app again."

**Device**
Device: `LG-H870S`
Android: `8.0.0`
Locale: `pt_BR`
Fritter: `v3.0.0-beta6`

**Comment by GET100PERCENT** [comment]
I am on Android 7 Nougat.
I am also facing the same issue.

**Comment by GET100PERCENT** [comment]
https://user-images.githubusercontent.com/116144024/208069224-d57ab8a7-e72d-41a3-8ff6-7dd4e0a5cdd3.mp4

**Comment by jonjomckay** [comment]
I'm not able to reproduce this 🙁 

If it's possible, the output from `logcat` when this happens would be super useful in figuring out what's broken!

**Comment by GET100PERCENT** [comment]
How to get the logcat

**Comment by autogris** [comment]
I'm experiencing the same problem since the first beta release on an Android 8.1.0. Here is a logcat file filtered for "fritter", trying to run beta-7 and crashing as soon the window is drawn.

[logcat_04-23-2023_14-52-41.txt](https://github.com/jonjomckay/fritter/files/11311066/logcat_04-23-2023_14-52-41.txt)

**Comment by jonjomckay** [comment]
Thanks for the logcat output @autogris. Looking at the stack traces, it appears the problem comes from KDE Connect, not Fritter. Can you try disabling KDE Connect temporarily and seeing if it fixes the issue?

Does anyone else on this issue also have KDE Connect installed, or some other app that reads notifications?

**Comment by GET100PERCENT** [comment]
> Thanks for the logcat output @autogris. Looking at the stack traces, it appears the problem comes from KDE Connect, not Fritter. Can you try disabling KDE Connect temporarily and seeing if it fixes the issue?
> 
> Does anyone else on this issue also have KDE Connect installed, or some other app that reads notifications?

No

**Comment by autogris** [comment]
hi @jonjomckay, thanks for checking this. I uninstalled KDE Connect and took a new log this time with Beta 8. The crashing persists (I see there is a mention of gadgetbridge but I think this is also unrelated, I have just installed this app and the Fritter crash is happening since last year).

[logcat_04-29-2023_15-04-40.txt](https://github.com/jonjomckay/fritter/files/11359804/logcat_04-29-2023_15-04-40.txt)
